src/modules/API.js

```diff
@@ -6,9 +6,8 @@ export const searchByName = async (Name) => {
     const data = await response.json();
     return data;
   } catch (error) {
-    console.error(error);
+    throw new Error('Something went wrong!');
```

Comment:
It's great to change the **console log** and have a more friendly **error** but using the `error` from `catch(error)` would be great to add in there
Example :  throw new Error('Something went wrong!', error );